MetroIntentsUI/IntentViewController.swift

```diff
@@ -0,0 +1,61 @@
+import IntentsUI
+import SwiftUI
+import MetroKit
+import MetroUI
+
+class IntentViewController: UIViewController, INUIHostedViewControlling {
+    let padding: CGFloat = 8
+
+    // MARK: - INUIHostedViewControlling
+
+    var desiredSize: CGSize {
+        let maxWidth = self.extensionContext!.hostedViewMaximumAllowedSize.width
```

Comment:
Do you want to subtract `2 * padding` here?

Reply:
I ended up making the padding proportional in the SwiftUI view directly, so we don't need to add it here.